File: Week-02/Day-02/seconds-in-a-day.py
current_hours = 14;
current_minutes = 34;
current_seconds = 42;

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented bt the variables

TotalDaySec = 24*60*60
Hours = 14*60*60
Mins = 34*60
Secs = 42

print (TotalDaySec)
print (Hours)
print (Mins)

print ("Remaining seconds from the day: " + str (TotalDaySec - (Hours + Mins + Secs)))

File: Week-07/Day-02/seconds-in-a-day.js

'use strict';

var currentHours = 14;
var currentMinutes = 34;
var currentSeconds = 42;

// Write a program that prints the remaining seconds (as an integer) from a
// day if the current time is represented by these variables

var TotalDaySec = 24*60*60;
var Hours = 14*60*60;
var Mins = 34*60;
var Secs = 42;

console.log(TotalDaySec);
console.log(Hours);
console.log(Mins);

console.log("Remaining seconds from the day: " + (TotalDaySec - (Hours + Mins + Secs)));
